src/libgit2/CMakeLists.txt

```diff
@@ -1,9 +1,18 @@
 # libgit2: the shared library: this CMakeLists.txt compiles the core
 # git library functionality.
 
-add_library(libgit2 OBJECT)
-set_target_properties(libgit2 PROPERTIES C_STANDARD 90)
-set_target_properties(libgit2 PROPERTIES C_EXTENSIONS OFF)
+file(GLOB SRC_GIT2 *.c *.h
```

Comment:
Why glob _here_?

Reply:
this GLOB should dissappear (cf. previous comments)